insights/parsers/jboss_standalone_main_conf.py

```diff
@@ -0,0 +1,109 @@
+"""
+JBoss standalone mode main configuration - file ``$JBOSS_BASE_DIR/standalone.xml``
+==================================================================================
+
+This parser reads the XML in the JBoss standalone mode main configuration file and provides a
+standard ElementTree interface to it.  It also provides a ``get_properties`` function
+that lists all the required properties.
+
+Sample input::
+
+    <?xml version='1.0' encoding='UTF-8'?>
+
+    <server xmlns="urn:jboss:domain:1.7">
+        <management>
+            <security-realms>
+                <security-realm name="ManagementRealm">
+                    <authentication>
+                        <local default-user="$local" skip-group-loading="true"/>
+                        <properties path="mgmt-users.properties" relative-to="jboss.server.config.dir"/>
+                    </authentication>
+                    <authorization map-groups-to-roles="false">
+                        <properties path="mgmt-groups.properties" relative-to="jboss.server.config.dir"/>
+                    </authorization>
+                </security-realm>
+                <security-realm name="ApplicationRealm">
+                    <authentication>
+                        <local default-user="$local" allowed-users="*" skip-group-loading="true"/>
+                        <properties path="application-users.properties" relative-to="jboss.server.config.dir"/>
+                    </authentication>
+                    <authorization>
+                        <properties path="application-roles.properties" relative-to="jboss.server.config.dir"/>
+                    </authorization>
+                </security-realm>
+            </security-realms>
+        </management>
+    </server>
+
+Examples:
+
+    >>> from insights.tests import context_wrap
+    >>> from insights.parsers.jboss_standalone_main_conf import JbossStandaloneConf
+    >>> JBOSS_STANDALONE_CONFIG = '''
```

Comment:
This is fine, but I'd recommend following the current practices for example testing explained in https://github.com/RedHatInsights/insights-core/pull/845 - you don't need to re-include this content or do the environment set-up within the documentation.  It also allows you to do some of the standard testing of properties, etc. in your examples rather than duplicate that in the test suite.

Reply:
@PaulWay Good suggestion!